tests/integration/recipe/test_check.py

```diff
@@ -1,13 +1,14 @@
-"""Integration tests for :mod:`esmvalcore._recipe_checks`."""
```

Comment:
I'd keep the original test module name `test_recipe_checks`

Reply:
That would be confusing because module is no longer called that, it has been moved to [esmvalcore/_recipe/check.py](https://github.com/ESMValGroup/ESMValCore/blob/add-dataset/esmvalcore/_recipe/check.py)

Reply:
OK up to you, bud